stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -79,6 +79,66 @@ var result = websub:startHub(new http:Listener(8080), hubConfiguration = hubConf
 
 Any subscriptions added at the hub will now be available even when the hub is restarted.
 
+##### Enabling Basic Auth support for the hub
+
+The Ballerina WebSub Hub can be secured using basic authentication by enforcing authorization. Hub listener and the 
+Hub service need AuthProvider and authConfig configurations respectively. The auth provider stores username, password
```

Comment:
`AuthProvider` and `authConfig` need to be specified for the hub listener and service respectively.

If the `authStoreProvider` of the `AuthProvider` is set as "config", usernames and passwords for authentication and scopes for authorization would be read from a config toml file.

A user can specify `AuthProvider` as follows and set it to the `hubListenerConfig` record passed when starting the hub.